doc/source/installation/windows_wsl.rst

```diff
@@ -234,21 +234,25 @@ Do not forget the ``.`` at the end of the command, which means "here".
-  +--------+------------------------------------------------+-----------------------------------------------+
-  |     12 | ``CHECKSUM="${CHECKSUM} 0ea6e4806b6...``       | ``#CHECKSUM="${CHECKSUM} 0ea6e4806b6...``     |
-  +--------+------------------------------------------------+-----------------------------------------------+
+    The prefix ``#`` is used to comment a line. Here we are commenting lines 9 to 12 and adding 2 new lines to change the p4est version.
+
+  +--------+------------------------------------------------+-------------------------------------------------------------------------------+
+  | line # | initial line                                   | changed line                                                                  |
+  +========+================================================+===============================================================================+
+  |     9  | ``VERSION=2.3.2``                              | ``#VERSION=2.3.2``                                                            |
+  +--------+------------------------------------------------+-------------------------------------------------------------------------------+
+  |     10 | ``CHECKSUM=076df9e...``                        | ``#CHECKSUM=076df9e...``                                                      |
+  +--------+------------------------------------------------+-------------------------------------------------------------------------------+
+  |     11 | ``CHECKSUM="${CHECKSUM} b41c8ef29ca...``       | ``#CHECKSUM="${CHECKSUM} b41c8ef29ca...``                                     |
+  +--------+------------------------------------------------+-------------------------------------------------------------------------------+
+  |     12 | ``CHECKSUM="${CHECKSUM} 0ea6e4806b6...``       | ``#CHECKSUM="${CHECKSUM} 0ea6e4806b6...``                                     |
+  +--------+------------------------------------------------+-------------------------------------------------------------------------------+
+  |     13 |                                                | .. code-block:: text                                                          |
+  |        |                                                |   :class: copy-button                                                         |
+  |        |                                                |                                                                               |
+  |        |                                                |   VERSION=2.3.6                                                               |
+  |        |                                                |   CHECKSUM=4b35d9cc374e3b05cd29c552070940124f04af8f8e5e01ff046e39833de5e153   |
+  +--------+------------------------------------------------+-------------------------------------------------------------------------------+
```

Comment:
I like this format for adding new lines. I suggest we write it the same way for regular installation and Apple ARM

Reply:
I was hesistating between both format, so I thought leaving them both here, the reviewers will tell me their opinion.